client/app/lib/flux/teams/actions.coffee

```diff
@@ -347,6 +348,21 @@ fetchCurrentStateOfApiAccess = ->
   state = team.isApiEnabled is yes
   reactor.dispatch actions.SET_API_ACCESS_STATE, { state }
 
+fetchOtaToken = ->
```

Comment:
we need to write this down into somewhere because I'm seeing it's mixed in this file already. https://github.com/koding/koding/pull/8659/files#diff-a4b7c9664d2e74d36a97a8137571b70eR390 Btw, I agree with `loadOtaToken` for this case, because even it's fetching something it's main purpose here to load some state into some store, we're not actually using this method to fetch the data.